src/pages/token-metadata/update.md

```diff
@@ -57,3 +125,56 @@ await updateV1(umi, {
 {% /totem %}
 {% /dialect %}
 {% /dialect-switcher %}
+
+### pNFT Asset
+
+This example shows you how to update a Programable NFT (pNFT) Asset as the update Authority of the Asset.
+
+{% dialect-switcher title="pNFT Asset Update" %}
+{% dialect title="JavaScript" id="js" %}
+{% totem %}
+
+```ts
+import { getMplTokenAuthRulesProgramId } from '@metaplex-foundation/mpl-candy-machine'
+import {
+  collectionToggle,
+  fetchMetadataFromSeeds,
+  TokenStandard,
+  updateAsUpdateAuthorityV2,
+} from '@metaplex-foundation/mpl-token-metadata'
+import { publicKey, unwrapOptionRecursively } from '@metaplex-foundation/umi'
+
+// The Mint ID of the pNFT Asset
+const mintId = publicKey('HckzL4iRK3j6DsznzedwNFzmFj4z7xCBoRkMMrd8THd5')
```

Comment:
Should this just be like '11111111' like your other examples rather than a real Pubkey?